app/models/rule.rb

```diff
@@ -98,7 +83,9 @@ def self.from_openscap_parser(op_rule, existing: nil, rule_group_id: nil, benchm
     rule.assign_attributes(title: op_rule.title, description: op_rule.description,
                            rationale: op_rule.rationale, severity: op_rule.severity,
                            precedence: precedence, rule_group_id: rule_group_id,
-                           upstream: false, slug: rule.normalize_friendly_id(op_rule.id))
+                           upstream: false, slug: rule.normalize_friendly_id(op_rule.id),
+                           identifier: op_rule.identifier.to_h,
```

Comment:
This makes `identifier` to be set as `{"label": null, "system": null}` instead of `nil`

```
-[ RECORD 8 ]-------------------------
id                    | 3b4df0ce-6a8a-4e2a-9a39-979e5dc7796e
ref_id                | xccdf_org.ssgproject.content_rule_sebool_cobbler_use_cifs
supported             | <null>
title                 | Disable the cobbler_use_cifs SELinux Boolean
severity              | medium
description           | By default, the SELinux boolean cobbler_use_cifs is disabled. If this setting is enabled, it should be disabled. To disable the cobbler_use_cifs SELinux boolean, run the following command: $ sudo setsebool -P cobbler_use_cifs off
rationale             |
created_at            | 2022-11-30 10:07:11.590838
updated_at            | 2022-11-30 10:07:11.590838
slug                  | xccdf_org-ssgproject-content_rule_sebool_cobbler_use_cifs
remediation_available | False
benchmark_id          | fa5976a3-3f07-4471-9a4b-3486d630d08e
upstream              | False
precedence            | 888
rule_group_id         | 05ef0523-1694-4f5d-940a-56e215c5d198
references            | []
identifier            | {"label": null, "system": null}
```

Reply:
https://github.com/OpenSCAP/openscap_parser/pull/45

Reply:
+version bump and update the dep

Reply:
I tested this with the openscap parser update in the comment above and it looks like that fixed the issue 👍 The `identifier` is correctly set to `nil` instead of `{"label"=>nil, "system"=>nil}`